docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc

```diff
@@ -5,6 +5,8 @@ Returns all user activity for the specified case.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
+WARNING: This endpoint is deprecated and will be removed in future releases.
```

Comment:
If we can't offer a concrete statement about when the endpoint will be removed, can we state the last version it's supported on?

Reply:
@nastasha-solomon Ok! Any suggestions? Thanks!

Reply:
If there are no legal constraints or other limitations to consider, maybe something like, "this endpoint is not available after version 8.1" or "this endpoint is available until version 8.1".

Reply:
> If there are no legal constraints or other limitations to consider
I am not aware of any. Do you have something in mind?

Reply:
Gotcha. Just recommending a minor edit then! Thanks again for your work on this!